src/sage/algebras/lie_algebras/bgg_dual_module.py

```diff
@@ -202,15 +202,12 @@ def degree_on_basis(self, m):
             sage: M = g.verma_module(La[1] + La[4] - 1/3*La[5])
             sage: Mc = M.dual()
             sage: elt = Mc.an_element(); elt
-            f[-alpha[5]]^2*f[-alpha[4]]^2*f[-alpha[3]]^3*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^*
-             + 2*f[-alpha[5]]*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^*
-             + 3*f[-alpha[4]]*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^*
-             + v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^*
+            f[-alpha[2]]^2*f[-alpha[5]]^2*f[-alpha[3]]^3*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^* + 2*f[-alpha[2]]*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^* + 3*f[-alpha[5]]*v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^* + v[Lambda[1] + Lambda[4] - 1/3*Lambda[5]]^*
```

Comment:
can you put this output on multiple lines, similarly to what it was before.

Reply:
done in e22d5f1f335d23fcdc5e6d7df40d35750339634e